packages/office-ui-fabric-react/src/components/Calendar/Calendar.types.ts

```diff
@@ -7,7 +7,7 @@ export interface ICalendar {
   focus: () => void;
 }
 
-export interface ICalendarProps extends IBaseProps<ICalendar> {
+export interface ICalendarProps extends IBaseProps<ICalendar>, React.HTMLAttributes<HTMLElement> {
```

Comment:
Wondering why this is `HTMLElement` vs `HTMLDivElement`? Not that it matters much, but aren't both components wrapped by a top-most `<div>`?